geometry_msgs/PoseStamped Pose
int16 grey_button
int16 white_button
geometry_msgs/Vector3 Footpedal